<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDialog</class>
 <widget class="QDialog" name="AdminDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>363</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>363</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Please enter admin &amp;password:</string>
     </property>
     <property name="buddy">
      <cstring>pwdEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pwdEdit">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Volume:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="leaveFullscreenBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Leave fullscreen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playPauseBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Play/Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Next track</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Clear playlist</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Quit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pwdEdit</tabstop>
  <tabstop>leaveFullscreenBtn</tabstop>
  <tabstop>playPauseBtn</tabstop>
  <tabstop>nextBtn</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>quitBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>cancelDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>quitApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwdEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>leaveFullscreenBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leaveFullscreenBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>leaveFullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>nextTrack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>clearPlaylist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPauseBtn</sender>
   <signal>clicked()</signal>
   <receiver>AdminDialog</receiver>
   <slot>playPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>AdminDialog</receiver>
   <slot>changeVolume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quitApp()</slot>
  <slot>cancelDialog()</slot>
  <slot>settings()</slot>
  <slot>leaveFullscreen()</slot>
  <slot>nextTrack()</slot>
  <slot>clearPlaylist()</slot>
  <slot>playPause()</slot>
  <slot>changeVolume(int)</slot>
 </slots>
</ui>
